libs/vkd3d/vkd3d_private.h

```diff
@@ -4885,6 +4932,32 @@ UINT d3d12_determine_shading_rate_image_tile_size(struct d3d12_device *device);
 bool d3d12_device_supports_required_subgroup_size_for_stage(
         struct d3d12_device *device, VkShaderStageFlagBits stage);
 
+static inline void d3d12_device_register_swapchain(struct d3d12_device* device, struct dxgi_vk_swap_chain* swapchain)
+{
+    spinlock_acquire(&device->low_latency_swapchain_spinlock);
+
+    if (!device->swapchain_info.low_latency_swapchain && device->swapchain_info.swapchain_count == 0)
+        device->swapchain_info.low_latency_swapchain = swapchain;
```

Comment:
Registering the low latency swapchain and removing must add/remove a private-ref on the swapchain.

Reply:
Swapchain registration, and removal each add/remove a private ref now.